### ISSUE TITLE
Add support for NixOS please

### ISSUE THREAD

**Issue by alxpettit** [cryinkfly/Autodesk-Fusion-360-for-Linux#309]
**Is your feature request related to a problem? Please describe.**

I'm trying to install on NixOS. The scripts, of course, do not work, and neither does the Bottle.

**Describe the solution you'd like**

I'd love if this script would add support for NixOS.

**Describe alternatives you've considered**

**Additional context**

NixOS is not a FHS system -- so it has problems with a lot of shell scripts. But its declarative configuration system gives it the potential to handle finicky setups like this more reliably than most others.

**Comment by 9ary** [comment]
I've got it working and I'm looking into putting together a proper NixOS setup, but I've been caught up in patching various projects to fix bugs that hurt usability. I'll try to remember to post my configuration here.

**Comment by alxpettit** [comment]
Um, any chance you could still post it? It would be a huge help! @9ary

**Comment by 9ary** [comment]
Sorry, I have nothing yet.

**Comment by Maybegus** [comment]
Any updates on this?

**Comment by 9ary** [comment]
Yes and no. The script in this repo can run an imperative installation, but there's no declarative setup that I'm aware of. Since I was never able to get fusion to run satisfactorily, I didn't bother writing my own.

**Comment by cryinkfly** [comment]
Can someone testing the new alpha installer? Or look to the code: https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/blob/main/files/setup/autodesk_fusion_installer_x86-64.sh

**Comment by alkeryn** [comment]
Hey, it's not a perfect fix but you could use an lxd container in the meanwhile, works for me at least.

**Comment by zentag** [comment]
I really need this as well.

**Comment by ImUrX** [comment]
> Can someone testing the new alpha installer? Or look to the code: https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/blob/main/files/setup/autodesk_fusion_installer_x86-64.sh

it fails, its also missing the gettext binary, i put it by hand to have text:
[error.txt](https://github.com/user-attachments/files/17998285/error.txt)
